ceph/rados/core_workflows.py

```diff
@@ -1498,6 +1498,31 @@ def fetch_host_node(self, daemon_type: str, daemon_id: str = None) -> object:
             )
             return None
 
+    def fetch_host_config(self, daemon_type: str, daemon_id: str = None) -> object:
+        """
+        Fetches the config of the given daemon in a cluster
```

Comment:
Please update the docstring to say what the method does. Also, the method name is ambiguous. Let's update the name something like : `get_daemon_config` or something similar.

Reply:
Please create both the modules under class `MonConfigMethods` in `tests.rados.monitor_configurations class MonConfigMethods`